postman_tests/lib/es.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import absolute_import
+import os
+import glob
+import json
+import csv
+import requests
+from . import common
+
+
+def get_es_api_url(config_path):
+    if os.path.exists(f"{config_path}/dev.json"):
+        config_file_path = f"{config_path}/dev.json"
+    else:
+        config_file_path = f"{config_path}/prod.json"
+    with open(config_file_path, 'r') as f:
+        data = json.load(f)
+    base_url = data["es_host"]
+    return f"http://{base_url}/entities/data/v1"
+
+
+def index_data(es_data_path, config_path):
+    """
+    Index data for every entity being tested into ElasticSearch
+
+    Parameters:
+    es_data_path (string): Path to the data/elastic_search directory
+
```

Comment:
Putting this once, but the docstrings everywhere have wrong format, indentation and wrong types.
Please read the guide at https://www.sphinx-doc.org/en/1.5/ext/example_google.html and fix docstrings.